pkg/cri/util/image.go

```diff
@@ -31,3 +39,8 @@ import (
 func NormalizeImageRef(ref string) (docker.Named, error) {
 	return docker.ParseDockerRef(ref)
 }
+
+// GetSandboxImageLabels returns label applied on sandbox image
+func GetSandboxImageLabels() (string, string) {
+	return sandboxImageLabelKey, sandboxImageLabelValue
+}
```

Comment:
Is there any value in specifying this function or would it be just easier and more readable to export (and use) the consts directly(?) 🤔 And, should they be specified in `labels/labels.go` instead?

A helper function like this would make more sense to me:
```suggestion
func IsSandboxImage(labels map[string]string) bool {
	return labels[sandboxImageLabelKey] == sandboxImageLabelValue
}
```